lib/views/home/home_view.dart

```diff
@@ -206,8 +227,8 @@ class _HomeViewState extends State<HomeView> {
                               child: Text(
                                 checkListIdentifier(
                                         listTile: snapshot.data![index])
-                                    ? snapshot.data![index].label
-                                    : snapshot.data![index].bolt11,
+                                    ? "Amount received : ${snapshot.data![index].amount}"
+                                    : "Amount Sent : ${snapshot.data![index].amountSent}",
```

Comment:
```suggestion
                                    ? " ${snapshot.data![index].amount}"
                                    : "- ${snapshot.data![index].amountSent}",
```

In my bank account, there is no `euro sent: xxxx` but there is the - in front this is much real.

I think there are a lot of examples of this use case in the app that we use every day, why we do not steal some ideas?

Reply:
Changed it. Also changed the color of the transactions from white to green or red depends upon the transaction